src/navbar.js

```diff
@@ -0,0 +1,46 @@
+import menu from './menu';
+import home from './home';
+import contact from './contact';
+
+const navbar = () => {
+  // navbar
+  const content = document.getElementById('content');
+  const nav = document.createElement('DIV');
+  nav.className = 'navbar';
+  content.appendChild(nav);
+  const logo = document.createElement('IMG');
+  logo.setAttribute('src', './images/logo.png');
+  nav.appendChild(logo);
+  const navItems = document.createElement('DIV');
+  navItems.className = 'nav-items';
+  nav.appendChild(navItems);
+
+  const element = document.createElement('SPAN');
+  element.innerHTML = 'HOME';
+  element.addEventListener('click', () => {
+    content.innerHTML = '';
+    navbar();
+    home();
+  });
+  navItems.appendChild(element);
+
+  const element2 = document.createElement('SPAN');
+  element2.innerHTML = 'CONTACT US';
+  element2.addEventListener('click', () => {
+    content.innerHTML = '';
+    navbar();
+    contact();
+  });
+  navItems.appendChild(element2);
+
+  const element3 = document.createElement('SPAN');
+  element3.innerHTML = 'MENU';
+  element3.addEventListener('click', () => {
+    content.innerHTML = '';
+    navbar();
+    menu();
+  });
+  navItems.appendChild(element3);
```

Comment:
[Optional] We could eliminate a lot of code duplication in this module either by using `forEach` or by defining a custom function which sets up the nav items.